Cargo.toml

```diff
@@ -57,6 +57,10 @@ required-features = ["time", "uuid", "chrono"]
 name = "data_types_variant"
 required-features = ["time"]
 
+[[example]]
+name = "stream_arbitrary_format_rows"
+required-features = ["futures03"]
+
```

Comment:
We also should provide an example for the use case "collect everything into one buffer"